src/Graphics/OpenGLContext/GLSL/glsl_CombinerProgramBuilder.cpp

```diff
@@ -1234,6 +1234,10 @@ class ShaderFragmentMain : public ShaderPart
 			"  lowp float alpha1;				\n"
 			"  lowp vec3 color1, input_color;	\n"
 		;
+		m_part += "#define WRAP(x, low, high) mod((x)-(low), (high)-(low)) + (low) \n"; // Return wrapped value of x in interval [low, high)
+		// m_part += "#define WRAP(x, low, high) (x) - ((high)-(low)) * floor(((x)-(low))/((high)-(low)))  \n"; // Perhaps more compatible?
+		// m_part += "#define WRAP(x, low, high) (x) - ((high)-(low)) * (ceil(((x)-(low))/((high)-(low))) - 1.0) \n"; // Return wrapped value of x in interval (low,high]
+
```

Comment:
May be add the old implementation for reference? It can be useful to quick check that a problem is not caused by the new code.